demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -358,6 +361,32 @@ def check_readme_relative_url_paths(self, is_pack_readme: bool = False) -> list:
 
         return error_list
 
+    def verify_image_exist(self) -> bool:
+        """Validate README images are actually exits.
+
+        Returns:
+            bool: True If all image path's actually exist else False.
+
+        """
+        images_path = re.findall(
+            r"\.\./doc_files/[a-zA-Z0-9_-]+\.png",
+            self.readme_content,
+        )
+
+        for image_path in images_path:
+            if not os.path.isfile(
+                Path(
+                    PACKS_DIR,
+                    get_pack_name(self.file_path),
+                    image_path.removeprefix("../"),
+                )
+            ):
```

Comment:
please assign Path(...) into a variable, and check `if not (path.exists() and path.is_file()): ...`